tsconfig.json

```diff
@@ -82,6 +82,9 @@
       ],
       "integration-test/*": [
         "integration-test/*"
+      ],
```

Comment:
Please remove this and similar one for `integration-test`, these are not needed since we have `"baseUrl": "src"`